schema_based_on_diagram.sql

```diff
@@ -1 +1,51 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
 
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT REFERENCES patients(id),
+  status VARCHAR(255),
+  PRIMARY KEY(id)
+);
+
+CREATE INDEX patient_histories ON medical_histories(patient_id);
+
+CREATE TABLE treatment_history (
+  history_id INT REFERENCES medical_histories(id),
```

Comment:
- [ ] It would be better if you rename this `history_id` to `medical_history_id` so it can be more descriptive and reflect the name in the diagram